src/django_mysql/models/fields/fixedchar.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from typing import Any, cast
+
+from django.db.backends.base.base import BaseDatabaseWrapper
+from django.db.models import CharField
+
+from django_mysql.typing import DeconstructResult
+
+
+class FixedCharField(CharField):
+    def __init__(self, *args: Any, length: int, **kwargs: Any) -> None:
+        if not isinstance(length, int):
+            raise TypeError(
+                'Invalid length value "{length}". '
+                "Expected integer value.".format(length=length)
+            )
+
+        if length < 0 or length > 255:
+            raise ValueError(
+                'Invalid length value "{length}". '
+                "Length must be in the range of 0-255.".format(length=length)
+            )
```

Comment:
I think we should follow Django's example and perform this in a system check. And use an f string.